raiden-dapp/src/components/notification-panel/NotificationCard.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-card class="notification-card">
+    <div class="notification-card__details-wrapper">
+      <v-avatar
+        class="notification-card__details-wrapper__icon"
+        tile
+        size="80"
+        color="grey"
+      />
+      <div>
+        <v-card-title></v-card-title>
+        <v-card-subtitle></v-card-subtitle>
+      </div>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({})
+export default class NotificationCard extends Vue {}
```

Comment:
I am guessing we are not adding the properties as part of this PR? right

Reply:
No, we'll add them with the first notification task. :)

Reply:
Ok sounds good :+1: